llama_stack/cli/stack/run.py

```diff
@@ -65,6 +65,13 @@ def _add_arguments(self):
             type=str,
             help="Path to TLS certificate file for HTTPS",
         )
+        self.parser.add_argument(
+            "--runtime",
```

Comment:
It's consistent with the existing naming though. I'll wait for others to comment as well

Reply:
happy to either change both or just leave this as `image-type`, but I will say `image-type` doesn't make as much sense in this context

Reply:
I am switching back to image-type for simplicity. I think switching other flag names is out of scope for the PR